streams/src/test/java/org/apache/kafka/streams/kstream/internals/WindowedStreamPartitionerTest.java

```diff
@@ -143,7 +144,48 @@ public void testWindowedDeserializerNoArgConstructors() {
         windowedDeserializer1.close();
     }
     @Test
-    public void testWindowedDeserializerWindowSize() {
+    public void testWindowSerializeConfig() {
+        final Map<String, String> props = new HashMap<>();
+        // test key[value].serializer.inner.class takes precedence over serializer.inner.class
+        final WindowedDeserializer<StringSerializer> windowedDeserializer = new WindowedDeserializer<>();
+        props.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "host:1");
+        props.put(StreamsConfig.APPLICATION_ID_CONFIG, "appId");
+        props.put("key.deserializer.inner.class", "org.apache.kafka.common.serialization.StringDeserializer");
+        props.put("deserializer.inner.class", "org.apache.kafka.common.serialization.StringDeserializer");
```

Comment:
There is not reason to specify this twice. If you want to test that it work for both parameter names, it should be two different test, each testing one parameter name